x-pack/plugins/stack_alerts/server/alert_types/index_threshold/alert_type_params.test.ts

```diff
@@ -71,3 +71,185 @@ describe('alertType Params validate()', () => {
     return ParamsSchema.validate(params);
   }
 });
+
+export function runTests(schema: ObjectType, defaultTypeParams: Record<string, unknown>): void {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let params: any;
+
+  const CoreDefaultParams: Writable<Partial<CoreQueryParams>> = {
```

Comment:
This helper test function has been duplicated between the two plugins.
This is because both plugins have components using the same Api and identical schemas.
I couldn't think of a clean way of sharing this test code in both plugins, so chose to duplicate instead.